@shared/api/helpers.ts

```diff
@@ -13,22 +13,24 @@ declare global {
 }
 
 export const sendMessageToContentScript = (msg: {}): Promise<Response> => {
+  const MESSAGE_ID = Date.now();
```

Comment:
the issue before was that we were instantiating 2 message listeners. They would each post a message, but both instances would respond and terminate as soon as the first message came back. So, as soon as `loadAccount` responded, both listeners would terminate and there was nothing left to respond to `loadSettings`. 

This tags each instance with a unique id so they will only respond to messages that they sent